lisa/tools/git.py

```diff
@@ -153,3 +153,33 @@ def init_submodules(self, cwd: pathlib.PurePath) -> None:
             expected_exit_code=0,
             expected_exit_code_failure_message="error on init submodules.",
         )
+
+    def get_tag(
+        self,
+        cwd: pathlib.PurePath,
+        sort_by: str = "v:refname",
+        contains: str = "",
+        return_last: bool = True,
+    ) -> str:
+        sort_arg = ""
+        contains_arg = ""
+        if sort_by:
+            # git tag exposes various sort options, apply them if present
+            # default is sort by version, ascending
+            sort_arg = f"--sort={sort_by}"
+        if contains:
+            # git tag allows you to filter by a commit id, apply it is present.
+            contains_arg = f"--contains {contains}"
+
+        git_cmd = f"--no-pager tag {sort_arg} {contains_arg}"
+        tags = self.run(
+            git_cmd,
+            cwd=cwd,
+            expected_exit_code=0,
+            expected_exit_code_failure_message="Could not get latest tagged commit ID",
```

Comment:
The message should be changed. It's failed on get tags, not matter about latest tagged.

Reply:
Will change, testing rn